sites/firstmedia.com/firstmedia.com.channels.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site site="firstmedia.com">
+    <channels>
+        <channel lang="id" xmltv_id="ABCAustralia.au" site_id="251">ABC Australia</channel>
```

Comment:
```suggestion
        <channel lang="en" xmltv_id="ABCAustralia.au" site_id="251">ABC Australia</channel>
```

Most channels have the guide in English.

https://github.com/iptv-org/epg/blob/master/CONTRIBUTING.md#how-do-i-add-a-program-guide-for-the-channel